Decide/target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst

```diff
@@ -0,0 +1 @@
+/home/felix/Desktop/DD2480/assig1/DD2480-decide/Decide/src/main/java/DD2480/decide/App.java
```

Comment:
Will this work for all of us? It seems to me that this is your local file path.

Reply:
Yeah should not have added that folder